### PR TITLE
Update pl.md

### PR DESCRIPTION
**Pull request by AdrianUrsea**
Fix Bug 1243088 - Updating to the Firefox Privacy Notice

### CODE REVIEW

**Comment by peiying2** [comment]
bug fix 1243088

File: firefox_privacy_notice/pl.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Firefox automatycznie nawiązuje połączenia z nami i naszymi dostawcami usł
...
 
-	Miniatury to funkcja przeglądarki Firefox wyświetlana na stronach nowych kart. Aby udostępniać funkcję miniatur, Firefox przesyła do firmy Mozilla powiązane z nimi dane, takie jak liczba kliknięć i wyświetleń, adres IP użytkownika, informacje o ustawieniach regionalnych i dane miniatur (np. położenie i rozmiar siatki). W przeglądarce Firefox Beta pewne krótkoterminowe eksperymenty związane z Telemetrią (patrz wyżej) dla funkcji Miniatury mogą zbierać informacje o często odwiedzanych domenach.
+	Podpowiedzi wyszukiwania to funkcja, która pomaga znaleźć hasła, których często szukali inni użytkownicy. Podpowiedzi są dostarczane przez domyślną wyszukiwarkę (np. Google, Yahoo itp.), a nie przez Firefoksa. Jeśli używana domyślna wyszukiwarka obsługuje podpowiedzi, po włączeniu tej funkcji Firefox może wysyłać hasła wpisywane w pasku Awesome Bar lub w pasku wyszukiwania do domyślnej wyszukiwarki, by pobrać podpowiedzi. Podlega to Polityce prywatności domyślnej wyszukiwarki. [Więcej informacji o podpowiedziach wyszukiwania można znaleźć tutaj](https://support.mozilla.org/kb/use-popular-search-suggestions-firefox-search-bar), razem z informacjami, jak włączyć i wyłączyć tę funkcję.
```
Comment (splewako) [inline firefox_privacy_notice/pl.md]: What the heck "Awesome Bar" is?

File: firefox_privacy_notice/pl.md [hunk 2]
```diff
@@ -76,3 +76,5 @@ Jeśli użytkownik o to poprosi, Firefox łączy się także z firmą Mozilla,
...
 {: #addons }
+
+* **Powiadomienia wypychane**: Powiadomienia wypychane umożliwiają witrynom wysyłanie powiadomień i aktualizacji, jeśli użytkownik wyrazi na to zgodę. Aby otrzymywać powiadomienia, Firefox wysyła do firmy Mozilla informacje o witrynach, z których użytkownik zgodził się je otrzymywać. Informacje te przechowujemy w formie zanonimizowanej wraz z liczbą powiadomień, które wysyła użytkownikowi każda z witryn. Aby pomóc programistom lepiej wykorzystywać powiadomienia wypychane, Mozilla może udostępniać niektórym programistom zagregowane dane, w tym liczbę użytkowników odwiedzających witrynę, którzy zasubskrybowali jej powiadomienia wypychane lub anulowali ich subskrypcję. Powiadomieniami wypychanymi w Firefoksie można zarządzać, wykonując [te instrukcje](https://support.mozilla.org/kb/push-notifications-firefox).
```
Comment (splewako) [inline firefox_privacy_notice/pl.md]: What the heck "Powiadomienia wypychane" is?

File: firefox_privacy_notice/pl.md (hunk 1 above)
Comment (kyoshino) [inline firefox_privacy_notice/pl.md]: Awesome Bar is the marketing term of the location bar. IMO this document should avoid using it.

Comment (splewako) [inline firefox_privacy_notice/pl.md]: Thank you, I will fix this in incoming pr with fixes for Polish docs.

**Pull request merged**